Handwriting/HandwritingIntentCSSValue.md

```diff
@@ -88,21 +88,27 @@ Authors are used to the [recommended practice of adding touch-action: none](http
 
 ### Keywords and States
 
+ The `touch-action` attribute accepts the following keywords:
+> Value: `auto` | `none` | [ [ `pan-x` | `pan-left` | `pan-right` ] | [ `pan-y` | `pan-up` | `pan-down` ] ] | `manipulation`
+
 The `handwriting` keyword indicates whether an element and the element's descendants will allow handwriting input when supported by the user agent. Handwriting will only be allowed for an element when its computed `touch-action` includes the `handwriting` keyword. By default, `auto` and `manipulation` will include the `handwriting` keyword.
 
-#### Determining enablement
+Note that `touch-action` does not indicate that some actions should take precedence over others, so discerning which interaction the pointer events should trigger will be the responsibility of the User Agent. For example, differentiating between a _pan-*_ gesture and a _handwriting_ gesture if both are avaialable. 
+
+### Determining enablement
 
 All CSS properties have computed values for all elements. The enablement of handwriting in a given `element` can be determined by running the following steps:
 
 1. If the computed value for `touch-action` on `element` and all of its ancestors include either keyword `auto`, `handwriting`, or `manipulation`, **enable handwriting**.
 2. If the computed value for `touch-action` on `element` or any of its ancestors does not include either keyword `auto`, `handwriting`, or `manipulation`, **disable handwriting**.
 
-### Caveats
+### Caveats / Cons
 
 A few pain points have been brought up that are worth discussion:
 * Web pages that currently have the `touch-action` property set for different elements will lose the handwriting capabilities on this element even if they don't want to disable it. When the new keyword ships, the absence of the value will be interpreted as the author of the webpage intentionally disabling handwriting.
 * Authors that specify `touch-action: manipulation` will be enabling `handwriting`, even when they might not want the behavior enabled in their webpage. These authors would then need to update their webpages to explicitly mention which behaviors they want, i.e. : `touch-action: pan-x pan-y pinch-zoom`.
-
+*   Using `touch-action` restricts handwriting implementations to touch input devices (such as stylus and touch), even though a platform could support handwriting capabilities for other controls, like mouse pointer events.
+	* `touch-action` determines the behavior of touch input devices indistinctly of which device is being used, either touch or stylus. In the future, these input devices might be separated into two different CSS attributes to allow things like, say, enable panning with finger touch events and only enable handwriting with a stylus.  
```

Comment:
Instead of "indistinctly", I think "agnostic" might fit better.

```suggestion
	* `touch-action` determines supported behaviors of touch input devices agnostic of which device is being used, either touch or stylus. In the future, these input devices might be separated into two different CSS attributes to allow things like, say, enable panning with finger touch events and only enable handwriting with a stylus.  
```